java/src/main/java/system/gameplay/roleplay/DiceRoller.java

```diff
@@ -5,26 +5,53 @@
 import java.util.List;
 import java.util.Random;
 
-public class DiceRoller {
+public final class DiceRoller {
 
-    private final Random random;
+    private static final Random random = new Random();
 
-    public DiceRoller() {
-        random = new Random();
+    public static List<Integer> rollAbilityScores() {
+        //TODO: (A) This line is here so the compiler won't complain, remove it when you add the functionality for this method
+        ArrayList<Integer> rollAbilityScoresList = new ArrayList<>();
+        int topThreeSixSidedDie = topThreeSixSidedDie();
+        for (int i = 1; i <= 6; i++) {
+                rollAbilityScoresList.add(topThreeSixSidedDie);
+                topThreeSixSidedDie = topThreeSixSidedDie();
```

Comment:
Also do to the above implementation, you end up calling this method one extra time on the last run through your loop.